dev/populate_chef_backend_data.sh

```diff
@@ -58,7 +58,7 @@ do
   for file in *; 
   do
     # Extract the contents of the tar file
-    knife upload file -s $SERVER_URL/organizations/"org$i" --config-option cookbook_path=/home/ubuntu/cookbook
+    knife upload $file -s $SERVER_URL/organizations/"org$i" --config-option cookbook_path=/home/ubuntu/cookbook
```

Comment:
why is this change needed

Reply:
This is a minor bug found in a script used to upload cookbooks to org by dev team. I fixed and pushed in same PR.